llvm/include/llvm-c/CAS/PluginAPI_functions.h

```diff
@@ -164,13 +164,17 @@ LLCAS_PUBLIC llcas_digest_t llcas_objectid_get_digest(llcas_cas_t,
 /**
  * Checks whether a \c llcas_objectid_t points to an existing object.
  *
+ * \param globally if true it is a hint to the underlying implementation that
+ * the check should be done on a distributed caching level, not just locally,
+ * for implementations that have this kind of distinction.
  * \param error optional pointer to receive an error message if an error
  * occurred. If set, the memory it points to needs to be released via
  * \c llcas_string_dispose.
  * \returns one of \c llcas_lookup_result_t.
  */
 LLCAS_PUBLIC llcas_lookup_result_t llcas_cas_contains_object(llcas_cas_t,
                                                              llcas_objectid_t,
+                                                             bool globally,
```

Comment:
What's the plan to manage this ABI break?

Reply:
The plan is to not do anything about it since it's not used in production yet 🤷
By luck the combination of older clang and newer C plugin API change is not breaking build system tests.